steepshot_io/graph/views.py

```diff
@@ -39,110 +49,86 @@ def group_steem_golos(self, list_steem, list_golos):
             idx += 1
         return res
 
-    def get_posts_count_monthly(self, platform=None):
-
+    def _get_data(self, platform=None, reverse=True, data_x=None, data_y=None, name_url=None):
         if platform == 'steem':
-            res = requests.get(settings.REQUESTS_URL_STEEM['posts_count_monthly']).json()
-            res.reverse()
+            try:
+                res = requests.get(settings.REQUESTS_URL_STEEM[name_url]).json()
```

Comment:
Please move this code to separate method because you use the same code for steem and golos platform.